### PR TITLE
Porting guide

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Add porting guide entries from ansible/ansible's 2.10 porting guide as changelog fragments.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
changelog

### CODE REVIEW

**Comment by felixfontein** [comment]
CC @acozine @samccann

File: changelogs/fragments/letsencrypt.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+removed_features:
+- "The ``letsencrypt`` module has been removed. Use ``acme_certificate`` instead."
```
Comment (samccann) [inline changelogs/fragments/letsencrypt.yml:2]: Let's talk about this on #ansible-doc - we used to have these modules as links. @abadger - would it work if this was changed to R(acme_certificate_module) or something like that?

Comment (felixfontein) [reply changelogs/fragments/letsencrypt.yml:2]: This is for the changelog, it needs to use pure reStructuredText. When included in the docsite RST refs would work, but the changelog is (I think) often viewed as a text file on github (i.e. with basic RST formatting) - and then they won't work.

**Comment by felixfontein** [comment]
@samccann thanks for reviewing this!

**Pull request merged**